src/Admin.php

```diff
@@ -136,12 +137,21 @@ public static function post_action_label($post_id) {
       $discount_percentage = 0;
     }
 
+    $formatted_regular_price = wc_price($regular_price, ['price_format' => '%1$s&nbsp;%2$s']);
+    // Replace ",00" with ",-".
+    $formatted_regular_price = preg_replace('@,00@', ',–', $formatted_regular_price);
```

Comment:
Wouldn't be safer to use an HTML entity?
```php
$formatted_regular_price = preg_replace('@,00@', ',&mdash;', $formatted_regular_price);
```

Reply:
I don't know why exactly, but @sun said in a previous review: "Please use the Unicode character in the code, not the HTML entity."

Reply:
btw, we're using the endash, because the emdash looked way too long on the label.